### DISCUSSION TITLE
Newbie question

### DISCUSSION THREAD

**Discussion by cphil46** [netblue30/firejail#4505] [Q&A] [answered]
Hello, I'm new to firejail, so be patient with me, please.
As I started, all seemed well, until I considered looking at my /var/log/syslog from a firejailed kate editor.
Impossible. I tried to figure out why and it seemed to me that firejail used another /var directory.
Seems ok, even if I don't really understand how it works and why I can't access either /var/log/ since I'm in the adm group, after all.
After having fuzzed some neurons, I figured out that kate firejail profile was ok for normal, non-privegied users, but that I just doesn't wanted it for any admin task. I'm I right ?

By the way, I also noticed that firejailing kcalc broke the call of browser to display help seemingly not being able to determine wich default browser to call. Maybe related to my Desktop being pekwm. Any clue to how to arrange that should be welcome.

Have a nice day

**Comment by rusty-snake** [comment, marked as answer]
kate.profile includes `include whitelist-var-common.inc` which enables whitelisting for `/var`.
In order to access `/var/log` in a firejailed kate instance, you need to add `whitelist /var/log` to your kate.local (just create it at `~/.config/firejail/kate.local`; you need to use a unfirejailed editor).

If you need additional supplementary groups (`adm`), add `ignore nogroups` and `ignore noroot` to your kate.local.

**Reply by cphil46** [reply]
Thanks a lot.
Just to know. If I can do that, unprivileged users may do it too. That wouldn't help them much, only enabling them to "ls /var/log", no great deal. But in some other case that could be annoying. I don't wanna restrain their access to their own ~/.config/firejail, so what ?

**Reply by rusty-snake** [reply]
Sorry, I don't understand. What are you trying?

**Reply by cphil46** [reply]
I was wondering which good it is to restrict user's access if that same user has the possibility to give himself it's access back.
So must I restrict my user's ~/.local/firejail to read-only ?
I ask for I recently inherited of a specially foolish one. You know, the kind that "knows about computers".

**Reply by rusty-snake** [reply]
Do you use firejail to restrict users or programs?

**Reply by cphil46** [reply]
Right. I presume that you mean that firejail exists to restrict programs and restricting (or not) users is my responsibility.

**Reply by rusty-snake** [reply]
Actually you can use firejail to restrict users by setting it as their login-shell (maybe there are some backholes, systemd?). Inside a firejail sandbox you can not edit files in `~/.config/firejail`.

**Reply by cphil46** [reply]
Thanks for the hint. For now it seems a little extreme to me as long as I have not well familiarized myself with the concepts and their implementation. But someday, maybe. For now, I will just chown root:users ~/.config/firejail, mod 755.

**Comment by rusty-snake** [comment]
> I also noticed that firejailing kcalc broke the call of browser to display help seemingly not being able to determine wich default browser to call.

The good old "can not open links from firejailed program" issues ...

The best summaries are ATM in #3471 and #3919 AFAICTY.
